BarcodeHeroDemo/BarcodeHeroDemo/Info.plist

```diff
@@ -33,8 +33,6 @@
 	<key>UISupportedInterfaceOrientations</key>
 	<array>
 		<string>UIInterfaceOrientationPortrait</string>
-		<string>UIInterfaceOrientationLandscapeLeft</string>
```

Comment:
why are we deleting these?

Reply:
It doesn't support multiple orientations right now, will add it back in when it does I think.